### PR TITLE
Add community announcement for CVE-2019-12995 (#4581)

### PR DESCRIPTION
**Pull request by fpesce**
* Add commmunity announcement for CVE-2019-12995

* Address @geeknoid's comments

* Address linting

* Address linter bash issue

* Add needed spelling dictionary entries

* Address linting

* Fix linting

Please provide a description for what this PR is for.
cherrypick from #4581

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/blog/2019/cve-2019-12995/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: Needs front-matter.

File: content/blog/2019/cve-2019-12995/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+This bug affects all versions of Istio that are using the JWT authentication policy.
+
+The symptoms of the bug is an HTTP 503 error seen by the client, and
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: sympton, singular

File: content/blog/2019/cve-2019-12995/index.md (hunk 1 above)
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: Istio Proxy -> Envoy, here and elsewhere

File: content/blog/2019/cve-2019-12995/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+{{< text text >}}
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: text plain

File: content/blog/2019/cve-2019-12995/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Epoch 0 terminated with an error: signal: segmentation fault (core dumped)
+{{< /text >}}
+
+in the Envoy logs.
+
+The Envoy crash can be triggered using a malformed JWT without a valid signature, and on any URI being accessed regardless of the trigger\_rules in the JWT specification. Thus, this bug makes the Istio proxy vulnerable to a potential DoS attack.
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: The trigger\_rule is probably incorrect.

File: content/blog/2019/cve-2019-12995/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Impact score
+
+Overall CVSS score: 7.5 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:F/RL:O/RC:C](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:F/RL:O/RC:C)
+
+## Vulnerability impact and Detection
+
+The Istio proxy is vulnerable if the following two conditions are satisfied:
+* A JWT authentication policy is applied to it.
+* The JWT issuer (specified by `jwksUri`) uses RSA algorithm for signature verification (Note the bug is just triggered in this condition and have nothing to do with the RSA algorithm)
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: uses the RSA

Comment (geeknoid) [reply content/blog/2019/cve-2019-12995/index.md]: Have nothing -> has nothing

File: content/blog/2019/cve-2019-12995/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+This bug is fixed in the following versions of Istio:
+
+* For Istio 1.0.x deployments: update to a minimum version of Istio 1.0.9
+* For Istio 1.1.x deployments: update to a minimum version of Istio 1.1.10
+* For Istio 1.2.x deployments: update to a minimum version of Istio 1.2.2
+
+If you cannot immediately upgrade to one of these releases, you have the additional option of injecting a [`Lua` filter](https://github.com/istio/tools/tree/master/examples/luacheck) into older versions of Istio.   This filter has been verified to work with Istio versions 1.1.9, 1.0.8, 1.0.6, and 1.1.3.
+
+The `Lua` filter is injected *before* the Istio `jwt-auth` filter.
```
Comment (geeknoid) [inline content/blog/2019/cve-2019-12995/index.md]: This paragraph is completely out of place. What Lua filter, what does this have to do with the CVE?

File: content/blog/2019/cve-2019-12995/index.md (hunk 3 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]:
```suggestion
{{< text plain >}}
```

File: content/blog/2019/cve-2019-12995/index.md (hunk 1 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]:
```suggestion
A bug in Istio’s JWT validation filter causes Envoy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```

File: content/blog/2019/cve-2019-12995/index.md (hunk 4 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]: Some markdown linter do not like the _ to not be escaped, I wasn't sure about the markdown interpreter we're using. What would be the correct approach? `trigger_rules` ?

File: content/blog/2019/cve-2019-12995/index.md (hunk 5 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]:
```suggestion
* The JWT issuer (specified by `jwksUri`) uses the RSA algorithm for signature verification (Note the bug is just triggered in this condition and has nothing to do with the RSA algorithm)
```

File: content/blog/2019/cve-2019-12995/index.md (hunk 6 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]: This line refers to the previous one.

File: content/blog/2019/cve-2019-12995/index.md (hunk 2 above)
Comment (fpesce) [reply content/blog/2019/cve-2019-12995/index.md]: no, it's symptoms they are two (and thus, the real comment is : symptoms of the bug *are*).
503 from client + log line in Envoy.

**Pull request merged**